File: Introductory_Problems/CPP/Repetetions.cpp
#include<bits/stdc++.h>
using namespace std;


int main(){
    string str;
    cin >> str;
    int n = str.length();
    int ans = 0, cur = 0;
    char ch = str[0];
    for(int i = 0; i < n; i++){
        if(ch == str[i]){
            cur++;
        }else{
            cur = 1;
        }
        ch = str[i];
        ans = max(ans, cur);
    }
    cout << ans << endl;
    return 0;
}

File: Introductory_Problems/Python/Repetetions.py
def solution(n):
    ch = n[0]
    ans, cur = 0, 0

    l = len(n)
    for i in range(l):
        if n[i] == ch:
            cur = cur + 1
        else:
            cur = 1
        ch = n[i]
        ans = max(ans,cur)
    
    return ans

if __name__ == "__main__":
    n = input()
    print(solution(n), end = ' ')